domain/appeal.go

```diff
@@ -75,7 +75,7 @@ func (a *Appeal) Cancel() {
 func (a *Appeal) Activate() error {
 	a.Status = AppealStatusActive
 
-	if a.Options != nil && a.Options.Duration != "" {
+	if a.Options != nil && a.Options.Duration != "" && a.Options.Duration != "0h" {
```

Comment:
let's use `time.Duration` to check if the duration is zero

Reply:
Done. modified the logic accordingly

Reply:
also updated the documentation for valid duration values.